src/plots/scatterPlot.ts

```diff
@@ -133,6 +133,43 @@ export class Scatter<X, Y> extends XYPlot<X, Y> {
     return drawSteps;
   }
 
+  protected _propertyProjectors(): AttributeToProjector {
+    const propertyToProjectors = super._propertyProjectors();
+
+    propertyToProjectors["transform"] = this._constructTransformProjector();
+    if (this.renderer() === "svg") {
+      propertyToProjectors["d"] = this._constructSymbolProjector();
+    }
+
+    return propertyToProjectors;
+  }
+
+  protected _constructTransformProjector() {
+    const xProjector = Plot._scaledAccessor(this.x());
+    const yProjector = Plot._scaledAccessor(this.y());
+    return (datum: any, index: number, dataset: Dataset) => {
+      return this.renderer() === "svg"
+        ? "translate(" + xProjector(datum, index, dataset) + "," + yProjector(datum, index, dataset) + ")"
+        : { x: xProjector(datum, index, dataset), y: yProjector(datum, index, dataset) };
+    };
+  }
+
+  protected _constructSymbolProjector() {
+    const symbolProjector = Plot._scaledAccessor(this.symbol());
+    const sizeProjector = Plot._scaledAccessor(this.size());
+    return (datum: any, index: number, dataset: Dataset) => {
+      return symbolProjector(datum, index, dataset)(sizeProjector(datum, index, dataset));
+    };
+  }
+
+  protected _d3SymbolFactory() {
+    const symbolProjector = Plot._scaledAccessor(this.symbol());
+    const sizeProjector = Plot._scaledAccessor(this.size());
+    return (datum: any, index: number, dataset: Dataset, context: CanvasRenderingContext2D) => {
```

Comment:
This should be DRY-ed with _constructSymbolProjector I think. The implementation is exactly the same except we pass the `context` here, which doesn't make sense - why are we caring about the canvas rendering context in this place? We should try to keep setting/manipulating the "context" only in the drawer